apps/cookbook/src/app/showcase/label-showcase/label-showcase.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="example">
+  <p>
+    A kirby-label is a layout element used to structure text content inside
+    <a href="https://cookbook.kirby.design/#/home/showcase/item">Items.</a>
+    Recommended usage of Kirby-label and the standard label are illustrated in the following
+    examples. Kirby-label should be used in item components and label should be used in form fields.
+  </p>
```

Comment:
I think this text could be trimmed a bit, for better readability:
```suggestion
  <p>
    A kirby-label is a layout element used to structure text content inside
    <a href="https://cookbook.kirby.design/#/home/showcase/item">Items.</a>
    Recommended usage of kirby-label and its relation to other types of labels are illustrated in the following
    examples.
  </p>
```